dependency-check-suppressions.xml

```diff
@@ -1,25 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <suppressions
         xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.1.xsd">
-    <suppress>
-        <notes><![CDATA[
-   file name: tomcat-annotations-api-8.5.23.jar
-   ]]></notes>
-        <gav regex="true">^org\.apache\.tomcat:tomcat-annotations-api:.*$</gav>
-        <cpe>cpe:/a:apache:tomcat</cpe>
-    </suppress>
-    <suppress>
+        <suppress>
         <gav regex="true">.*</gav>
-        <cve>CVE-2015-2156</cve>
-        <cve>CVE-2014-3488</cve>
-        <cve>CVE-2018-11039</cve>
-        <cve>CVE-2018-11040</cve>
-    </suppress>
-    <suppress>
-        <notes><![CDATA[
-   file name: guava-.*
-   ]]></notes>
-        <gav regex="true">^com\.google\.guava:guava:.*$</gav>
-        <cpe>cpe:/a:google:guava</cpe>
+       <cve>CVE-2018-10237</cve>
```

Comment:
Do we still need this?

Reply:
Yup, basically app insights still has this vulnerability, they haven't release a new version that fixes the exploit but they've committed it, my guess is the fix is coming soon, so I'm ignoring the exploit itself not the whole dependency until they get their stuff together :D